weather/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+var path = require('path');
+module.exports = {
+    entry: './src/js/app.jsx',
```

Comment:
Always split your bundle into separate files. One would include the code for your app i.e. `app.js` and the other all dependencies i.e. `
vendor.js`